cpp/dolfinx/common/Timer.h

```diff
@@ -30,11 +31,9 @@ namespace dolfinx::common
 class Timer
 {
 public:
-  /// Create timer without logging
-  Timer();
-
-  /// Create timer with logging
-  Timer(const std::string& task);
+  /// Create timer
+  /// if a task name is provided this enables logging, otherwise without logging
```

Comment:
Add line break (this affects Doxygen formatting). Capitalise and punctuate. Sentence could be made clearer.